### PR TITLE
Sylius training proposal

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

File: source/_lectures/getting-started-with-sylius.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# max. počet lidí, který zvládneš a je ještě efektinví pro předávání tvého know-how
+# doporučujeme: 6-10
+capacity: "10 lidí"
+
+image: "/assets/images/logos/sylius.png"
+
+perex: "Comprehensive introduction to Sylius E-Commerce framework. Join us and learn how to customize one of the most promising e-commerce solution."
+---
+
+Have you heard about Sylius E-Commerce framework? If so, you know already that it is an important part of Symfony ecosystem and it is one of the biggest, publicly available projects developed in BDD methodology. If not, join me and check that e-commerce development can be a pleasure.
```
Comment (dukusz) [inline source/_lectures/getting-started-with-sylius.md]: you know already -> you already know

File: source/_data/users.yml [hunk 2]
```diff
@@ -43,3 +43,9 @@ parameters:
             name: "Ukázkový Školitel"
             url: "https://webkdesedovisomnevice.cz/"
             photo: "ukazkovy_skolitel.jpg"
+        118:
+            name: "Łukasz Chruściel"
+            url: "https://www.sylius.com/"
+            photo: "lukasz_chrusciel.jpg"
+            # for lectors
+            bio: "With Sylius from the early beginning, currently in the role of Software Engineer and Training Terminator. Kind of an API-guy, pragmatic developer in love with tests automatization and clean code."
```
Comment (dukusz) [inline source/_data/users.yml]: Training Terminator -> Trainer

File: source/_lectures/getting-started-with-sylius.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+# x999, obvykle 3 k za půldenní, 5 k za celodenní
+price: 4999
+
+# max. počet lidí, který zvládneš a je ještě efektinví pro předávání tvého know-how
+# doporučujeme: 6-10
+capacity: "10 lidí"
+
+image: "/assets/images/logos/sylius.png"
+
+perex: "Comprehensive introduction to Sylius E-Commerce framework. Join us and learn how to customize one of the most promising e-commerce solution."
```
Comment (dukusz) [inline source/_lectures/getting-started-with-sylius.md]: E-Commerce -> eCommerce or e-commerce

File: source/_lectures/getting-started-with-sylius.md (hunk 1 above)
Comment (dukusz) [inline source/_lectures/getting-started-with-sylius.md]: E-commerce -> e-commerce
you know already -> you already know
join me and check that -> join me and learn the e-commerce technology of the future. Forget legacy platforms like Magento or PrestaShop. Just enjoy being a developer again!

File: source/_lectures/getting-started-with-sylius.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+### Co je potřeba před školením
+
+- Installed PHP 7.1
+- Installed [PHPStorm](https://www.jetbrains.com/phpstorm/download/)
+- Setted up [Sylius-Standard edition](http://docs.sylius.com/en/latest/book/installation/index.html)
+- Basic Symfony knowladge
+
+<br>
+
+After training you should be familiar with Sylius eco system and you should be able to decide if Sylius suites your needs.
```
Comment (dukusz) [inline source/_lectures/getting-started-with-sylius.md]: with Sylius ecosystem -> with Sylius basics and will be able to start your first e-commerce project.

File: source/_lectures/getting-started-with-sylius.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Training will be done in english.
```
Comment (dukusz) [inline source/_lectures/getting-started-with-sylius.md]: english -> English

File: source/_lectures/getting-started-with-sylius.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+### Co si ze školení odneseš?
+
+- Being familiar with Sylius architecture and bundles separation.
+- Creating a CRUD API in minutes.
+- Creating a new resource with Sylius Resource Bundle.
+- How to smoothly generate Admin CRUD scaffolding with Sylius Grid Bundle.
+- Learning how to use State Machine component in Sylius
+- How to improve project development with fixtures generation
+- Sending emails dispatched by Sylius Resource Events
+
```
Comment (dukusz) [inline source/_lectures/getting-started-with-sylius.md:36]: Maybe add some BDD basics?

**Review by dukusz — CHANGES_REQUESTED** [review]

File: source/_lectures/getting-started-with-sylius.md (hunk 6 above)
Comment (lchrusciel) [reply source/_lectures/getting-started-with-sylius.md:36]: not enough time :(

**Pull request closed without merging**